modules/tensor_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -305,6 +344,24 @@ CrackFrontDefinition::initialSetup()
 void
 CrackFrontDefinition::initialize()
 {
+  // Update the crack front for fracture integral calculations
+  // This is only useful for 3D growing cracks, which are described by the mesh cutter at this
+  // moment
+  if (_use_mesh_cutter)
+  {
+    _crack_front_points =
+        _crack_front_points_provider->getCrackFrontPoints(_num_points_from_provider);
+    _crack_plane_normals =
+        _crack_front_points_provider->getCrackPlaneNormals(_num_points_from_provider);
+    updateCrackFrontGeometry();
```

Comment:
Is there a way to maybe only do this call to `updateCrackFrontGeometry()` if the cutting mesh has changed since the last time?  My motivation for asking that is that it prints out the updated table every time that gets called, which might be excessive. Maybe we should consider not printing that out, but I find it helpful. If that's a big hassle, though, I think it's OK to leave it as-is for now.

Reply:
Done, which makes the screen output cleaner.  Please note that crack grows in all steps in current examples, so we still see the crack front Table printed out several times.  However, unnecessary printing-out has been resolved in this update.